src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -1804,5 +1804,237 @@ static void Main()
             CompileAndVerify(src1 + src2 + src3, expectedOutput: "C { Y = 22, X = 11, U = 44, Z = 33 }").VerifyDiagnostics();
             CompileAndVerify(new[] { src1, src2, src3 }, expectedOutput: "C { Y = 22, X = 11, U = 44, Z = 33 }").VerifyDiagnostics();
         }
+
+        [ConditionalFact(typeof(DesktopOnly))]
+        public void EqualityContractGetter_CompilerGeneratedAttribute()
+        {
+            var verifier = CompileAndVerify(@"
+using System;
+
+record C;
+", symbolValidator: validate);
+
+            verifier.VerifyTypeIL("C", @"
```

Comment:
>VerifyTypeIL [](http://example.com/codeflow?start=21&length=12)

It feels like the use of this method is an overkill. Especially if our goal is to simply check presence of ```CompilerGeneratedAttribute```, ```symbolValidator``` is good enough.